app/models/case/sar/offender.rb

```diff
@@ -41,11 +49,7 @@ def searchable_fields_and_ranks
     send_by_email: 'send_by_email',
   }
 
-  acts_as_gov_uk_date :date_of_birth,
-                      :date_responded,
-                      :date_draft_compliant,
-                      :external_deadline,
-                      :received_date
+  acts_as_gov_uk_date(*GOV_UK_DATE_FIELDS)
```

Comment:
Is this easier to read? Seems a little indirect, but not such that I'd complain about it. Ah - just read further down - we use the list again 👍 

But maybe we could have the `acts_as_gov_uk_date(*GOV_UK_DATE_FIELDS)` right after the frozen array so they're next to each other?

Reply:
I haven't verified that this fix works for all the existing places where the annoying error crops up